modules/bitgo/test/v2/unit/wallets.ts

```diff
@@ -1425,5 +1431,177 @@ describe('V2 Wallets:', function () {
       await wallets.acceptShare({ walletShareId: shareId });
       acceptShareNock.done();
     });
+
+    describe('bulkAcceptShare', function () {
+      const AssertionError = require('assert').AssertionError;
+
+      afterEach(function () {
+        nock.cleanAll();
+        nock.pendingMocks().length.should.equal(0);
+        sinon.restore();
+      });
+
+      it('should throw validation error for userPassword empty string', async () => {
+        try {
+          await wallets.bulkAcceptShare({ walletShareIds: [], userPassword: '' });
+          assert.fail('should have thrown the error');
+        } catch (err) {
+          err.should.deepEqual(new Error('Missing parameter: userPassword'));
```

Comment:
Dont use try catch for this. Just do `await wallets.bulkAcceptShare({ walletShareIds: [], userPassword: '' }).should.rejectedWith('<your err msg')`

Please do this for all your test cases